piptools/resolver.py

```diff
@@ -194,9 +200,8 @@ def __init__(
         unsafe_packages: set[str] | None = None,
     ) -> None:
         """
-        This class resolves a given set of constraints (a collection of
-        InstallRequirement objects) by consulting the given Repository and the
-        DependencyCache.
+        Make sure the legacy resolver is enabled and no backtracking resolver
```

Comment:
This new phrasing feels weird. That's not what the initializer seems to do...

Reply:
I have no expertise in writing docstrings for `__init__` functions. What would you expect it to contain? A listing of all the parameters? 

>  That's not what the initializer seems to do...

Well, it _does_ raise `PipToolsError` if the user didn't enable the legacy resolver, and removes an entry for the 2020 resolver from the enabled features?